src/audio/copier/copier.c

```diff
@@ -51,6 +54,10 @@ SOF_DEFINE_REG_UUID(copier);
 
 DECLARE_TR_CTX(copier_comp_tr, SOF_UUID(copier_uuid), LOG_LEVEL_INFO);
 
+#if CONFIG_MICROPHONE_PRIVACY
+static int mic_privacy_configure(struct comp_dev *dev, struct copier_data *cd);
+#endif
```

Comment:
why don't you add the mic_privacy_configure() so that forward declaration is not needed?
Plus:
```c
#else
static inline int mic_privacy_configure(struct comp_dev *dev, struct copier_data *cd) { return 0 }
#endif /* CONFIG_MICROPHONE_PRIVACY */
```
and you can save on the ifdefs in code